.github/workflows/ci_ubuntu.yml

```diff
@@ -17,18 +17,31 @@ jobs:
       matrix:
         config:
           - {
-              name: "Ubuntu Intel GCC",
+              name: "Ubuntu 24.04 Intel GCC",
               os: ubuntu-latest,
               cc: "gcc",
               cxx: "g++",
               cflags: "-msse2"
             }
           - {
-              name: "Ubuntu Intel Clang",
+              name: "Ubuntu 24.04 Intel Clang",
               os: ubuntu-latest,
               cc: "clang",
               cxx: "clang"
             }
+          - {
+              name: "Ubuntu 22.04 Intel GCC",
+              os: ubuntu-22.04,
+              cc: "gcc",
+              cxx: "g++",
+              cflags: "-msse2"
+            }
+          - {
+              name: "Ubuntu 22.04 Intel Clang",
```

Comment:
if the point is to get older lib coverage, i don't think we need to compile it twice

Reply:
Hmm, even if we assume that the libs are ABI compatible, we would need to transfer the build from one runner to another. So we would have to build on one runner, then upload to somewhere, only then start a dependent runner, download and run the test there. Looks like a serious complication, and might not even run faster.

But maybe I misunderstand what you have in mind.

Reply:
I mean we only need 1 Ubuntu 22 builder, so let's delete this one